update.sh

```diff
@@ -14,7 +14,26 @@ declare -A debianVariants=(
 	#[5.5]='jessie'
 )
 
+# Templating
+declare -A passwordset
+passwordset["5.6"]="DELETE FROM mysql.user WHERE user NOT IN ('mysql.sys', 'mysqlxsys', 'root') OR host NOT IN ('localhost') ;\nSET PASSWORD FOR 'root'@'localhost'=PASSWORD('\${MYSQL_ROOT_PASSWORD}');"
+passwordset["5.7"]="ALTER USER 'root'@'localhost' IDENTIFIED BY '\${MYSQL_ROOT_PASSWORD}';"
+passwordset["8.0"]="ALTER USER 'root'@'localhost' IDENTIFIED BY '\${MYSQL_ROOT_PASSWORD}';"
+declare -A database_init
+database_init["5.6"]='mysql_install_db --datadir="$DATADIR" --rpm --keep-my-cnf "${@:2}"'
+database_init["5.7"]='"$@" --initialize-insecure'
+database_init["8.0"]='"$@" --initialize-insecure'
+declare -A server_startup
+server_startup["5.6"]='"$@" --skip-networking --socket="${SOCKET}" \&'
+server_startup["5.7"]='"$@" --daemonize --skip-networking --socket="${SOCKET}" || result="$?"'
+server_startup["8.0"]='"$@" --daemonize --skip-networking --socket="${SOCKET}" || result="$?"'
```

Comment:
Given that we've already got blocks like `if [ "${MYSQL_MAJOR}" = "5.5" ] || [ "${MYSQL_MAJOR}" = "5.6" ]; then` in the entrypoint itself, wouldn't it make sense to move these in too and thus make the script completely standalone and ready to copy instead of it being a template?

(I think the resulting code could be slightly better too, since generating Bash typically results in a bit of odd flow, which I think we want to avoid here given that folks are likely to read these scripts especially if they want to use/extend them.)